build/elasticsearch/bin/es-docker

```diff
@@ -36,4 +35,28 @@ done < <(env)
 # will run in.
 export ES_JAVA_OPTS="-Des.cgroups.hierarchy.override=/ $ES_JAVA_OPTS"
 
+# Determine if x-pack platinum is enabled
```

Comment:
Correct, left over comment. We still need to ignore entirely this parameter is platinum is not installed, but I need to figure out another way to check this. TBD in a separate PR.